Prace_domowe/Praca_domowa4/Jamrozy_Bartosz/pd4.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>Jednym z kryteriów dobory drugiego zbioru danych mógłby być typ zadania.  W pierwszym była regresji, to w drugim mogłaby być klasyfikacja dla większej różnorodności. Niemniej jednak pomysł, aby przeanalizować kwestię predykcji cen nieruchomości z użyciem dwóch zbiorów danych, jest bardzo dobry.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/236/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>